.circleci/config.yml

```diff
@@ -14,8 +14,22 @@ parameters:
     type: enum
     enum: ["dss", "dsm","api-docs", "nothing"]
     default: "nothing"
+  action:
+    type: enum
+    enum: [ "build-test", "build-test-deploy", "build-test-store", "retrieve-deploy", "nothing" ]
+    default: "nothing"
 
 workflows:
+  build-test-deploy-dev-workflow:
+    when:
+      and:
+        - << pipeline.parameters.on_demand >>
+        - equal: ["dss", << pipeline.parameters.project >>]
+    jobs:
+      - continuation/continue:
+          configuration_path: .circleci/main-config.yml
+          parameters: '{"action":"build-test-deploy","dss": true,"api-docs": false}'
+
```

Comment:
Added this to CircleCi to allow me to deploy the PR branch to develop. It is difficult to test this problem locally.
(This is likely to stay as it can be very helpful).